content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -124,6 +124,25 @@ sudo mkdir -p /etc/containerd
 sudo containerd config default > /etc/containerd/config.toml
 ```
 
+```shell
+# Restart containerd
+sudo systemctl restart containerd
+```
+{{% /tab %}}
+{{% tab name="Ubuntu 18.04/20.04" %}}
+
+```shell
+# (Install containerd)
+## Install containerd
```

Comment:
👋  @karras 
I suppose you could remove the second comment, "Install containerd"
Question: Are the Docker steps required for 18.04/20.04? I have not tried to run these commands.

Reply:
Done, I've removed the second "Install containerd" comment. The Docker related steps are not required anymore as the containerd package from the official Ubuntu repositories is quite recent.

Reply:
OK